spec/spec_helper.rb

```diff
@@ -3,12 +3,19 @@
   SimpleCov.start
 end
 
-Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
-Dir[File.join(__dir__, "support/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].sort.each { |f| require f }
+Dir[File.join(__dir__, "support/**/*.rb")].sort.each { |f| require f }
 
 require "manageiq-providers-oracle_cloud"
 
 VCR.configure do |config|
   config.ignore_hosts 'codeclimate.com' if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::OracleCloud::Engine.root, 'spec/vcr_cassettes')
+
+  config.define_cassette_placeholder(Rails.application.secrets.oracle_cloud_defaults[:user_id]) do
+    Rails.application.secrets.oracle_cloud[:user_id]
+  end
+  config.define_cassette_placeholder(Rails.application.secrets.oracle_cloud_defaults[:tenant_id]) do
+    Rails.application.secrets.oracle_cloud[:tenant_id]
+  end
```

Comment:
Now that I look at it, should there be a `define_cassette_placeholder` for the public/private key as well?  I guess those aren't stored in the vcr casette?

Reply:
Those aren't stored in the cassette so we don't need placeholders for them